### DISCUSSION TITLE
Bypass Authentication and Authorization in iPLASS

### DISCUSSION THREAD

**Discussion by kyowanghwe** [dentsusoken/iPLAss#1665] [Q&A] [answered]
I want to write a API for testing, how can i bypass authen and author for specific API like in Spring
Thanks for your help !

**Comment by hkondoh** [comment, marked as answer]
Hi @kyowanghwe, thanks for using iPLAss.

To disable authentication and authorization control for each Web API, set `Privilege execute` to `true` .
This setting can be configured either in the Admin Console or by using the `@WebApi` annotation of the Command class.

![image](https://github.com/user-attachments/assets/9e5f039f-1068-480f-ba4d-4f4defd0deef)

[Web API Configuration](https://iplass.org/docs/developerguide/webapi/index.html#_%E5%9B%BA%E6%9C%89%E8%A8%AD%E5%AE%9A)
[WebApi Annotation](https://iplass.org/docs/developerguide/webapi/index.html#WebApi-Annotation)

If you don't want to change the access policy of the Web APIs just for testing, please consider creating a test-specific user and a role as an alternative.
While you cannot disable authentication for Web API calls, you can control user permissions for each Web API.

[Entity Permissions](https://iplass.org/docs/developerguide/authorization/index.html#en_entityperm)
[Web API Permissions](https://iplass.org/docs/developerguide/authorization/index.html#en_webapiperm)

**Reply by kyowanghwe** [reply]
Tks you so much for your help
